docs/src/test-advanced-js.md

```diff
@@ -301,6 +301,56 @@ test('test', async ({ page }) => {
 });
 ```
 
+You can make arbitary data available in your tests from your global setup file by attaching it to `process.env`.
+
+```js js-flavor=js
+// global-setup.js
+const { chromium } = require('@playwright/test');
+
+module.exports = async config => {
+  process.env.foo = 'some data';
```

Comment:
The names of environment variables are case-sensitive. By convention they are usually upper-case. Let's keep them also uppercase in this example?